src/lib.rs

```diff
@@ -315,6 +414,101 @@ macro_rules! call {
     };
 }
 
+macro_rules! add_event_handle_host {
```

Comment:
Can you rename this to `add_event_handler_host`(`handle` -> `handler`)? (And the same for the other two macros.)

Reply:
ok it's done